WordPress/src/test/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModelTest.kt

```diff
@@ -1379,17 +1481,26 @@ class ActivityLogViewModelTest {
     @Test
     fun `given status is a complete, when query backup status, then request events update for complete`() = test {
         val progress = BackupDownloadRequestState.Progress(REWIND_ID, 50)
-        val complete = BackupDownloadRequestState.Complete(REWIND_ID, DOWNLOAD_ID, DOWNLOAD_URL)
+        val complete = BackupDownloadRequestState.Complete(
+                REWIND_ID,
+                DOWNLOAD_ID,
+                DOWNLOAD_URL,
+                DOWNLOAD_PUBLISHED,
+                DOWNLOAD_VALID_UNTIL,
+                DOWNLOAD_IS_VALID)
         whenever(getBackupDownloadStatusUseCase.getBackupDownloadStatus(site, DOWNLOAD_ID))
                 .thenReturn(flow { emit(progress); emit(complete) })
         initBackupProgressMocks()
+        initBackupDownloadCompleteMocks()
         whenever(store.fetchActivities(anyOrNull()))
                 .thenReturn(OnActivityLogFetched(10, false, ActivityLogAction.FETCH_ACTIVITIES))
 
-        viewModel.onQueryBackupDownloadStatus(REWIND_ID, DOWNLOAD_ID)
+        viewModel.onQueryBackupDownloadStatus(REWIND_ID, DOWNLOAD_ID, JetpackBackupDownloadActionState.COMPLETE.id)
 
+        assertThat(viewModel.events.value?.first() as? Notice).isNotNull
```

Comment:
@ParaskP7 - This may not be ideal, but it gets the job done - for now. I'm having trouble creating a Notice item to match what is actually created because of the click delegates. I'm open to suggestions, but it can wait after the next beta. Thanks

Reply:
👋 @zwarm !

Definitely, will also have a quick look. 🙏